src/components/dialogs/export-dialog.js

```diff
@@ -67,64 +175,67 @@ const ExportDialog = ({
         setExportStudyErr('');
         setSelectedFormat('');
         setLoading(false);
-        setDownloadUrl('');
         onClose();
     };
 
-    const handleChange = (event) => {
+    const handleFormatSelectionChange = (event) => {
         let selected = event.target.value;
         setSelectedFormat(selected);
-        setDownloadUrl(getExportUrl(studyUuid, nodeUuid, selected));
     };
 
     const intl = useIntl();
 
     return (
         <Dialog
             fullWidth
-            maxWidth="xs"
+            maxWidth="sm"
             open={open}
             onClose={handleClose}
             aria-labelledby="dialog-title-export"
         >
-            <DialogTitle>{title}</DialogTitle>
+            <DialogTitle>
+                {title}
+                <FormControl fullWidth size="small">
```

Comment:
Put the Select component into the `DialogContent` instead of the `DialogTitle` which keep only the title then. This allow to have a padding between title dive and content div

Reply:
I want the scrolling area not to include the format selector nor the parameters fold open/close button